### PR TITLE
Revert "Added HTMLFormElement.requestSubmit()"

### PR DESCRIPTION
**Pull request by a2sheppy**
Reverts mdn/browser-compat-data#5488

This feature is disabled in the release of Firefox 73.

### CODE REVIEW

**Comment by a2sheppy** [comment]
Actually... this probably shouldn't just be reverted, but should revise the Firefox data. I didn't notice before submitting this that the PR actually added the missing item to the database. Let me tweak this PR and get back to you. :)

**Comment by ddbeck** [comment]
Hey Sheppy, I merged #5717 which updated the status data (I started on that PR before I saw yours because I like to welcome first-time contributors promptly). It might help to rebase or merge from `master` on this. The diff here is a bit mangled, but I get the impression that only the Firefox lines need to change now. Thank you for your help on this!

**Comment by stof** [comment]
the diff being big is because other lines have whitespace-only changes (there is a checkbox in the diff options on github to ignore whitespace changes, and this keeps only the firefox rows).

Also, this should not go back to `false` entirely. The feature is available behind a flag in 73+, not entirely missing.

**Comment by a2sheppy** [comment]
I... have no idea how this happened. Nor do I know how to fix it. I will have to do some research.

**Comment by vinyldarkscratch** [comment]
Oh wow, the diff is completely butchered, I honestly have no idea how that would happen...  (I also don't see any whitespace changes for those lines by the way; comparing their indentation shows no change.)  I would say it's probably best to start over on this one, considering the problematic diff and the needed rebase?

**Comment by stof** [comment]
@vinyldarkscratch I suspect tabs vs spaces, or line-ending changes.

**Comment by a2sheppy** [comment]
Let me see what I can figure out here...

**Comment by a2sheppy** [comment]
Huzzah!

File: api/HTMLFormElement.json [hunk 1]
```diff
@@ -764,7 +764,7 @@
           },
           "status": {
             "experimental": false,
-            "standard_track": true,
+            "standard_track": false,
```
Comment (stof) [inline api/HTMLFormElement.json]: why changing this ? It is on the standard track: https://html.spec.whatwg.org/multipage/forms.html#dom-form-requestsubmit-dev

**Pull request merged**